frontend/src/app/libs/common-components-demo/components/input-demo/input-demo.component.html

```diff
@@ -0,0 +1,79 @@
+<app-demo-group class="input-demo" [title]="'Input'">
+	
+    <div class="light-theme input-demo__states">
+		<div class="input-demo__row">
+			<form [formGroup]="inputTextDemoForm">
+				<app-demo-state [title]="'Text'" [isDark]="false">
+					<app-input
+						[inputValidators]="inputSyncValidators"
+						[inputType]="'text'"
+						[label]="'With input validate > 6 char'"
+						[placeholder]="'Your placeholder'"
+						formControlName="text1">
+					</app-input>
+					<app-input
+						[formControl]="inputTextDemoForm.controls['text2']"
+						[inputValidatorsAsync]="inputAsyncValidators"
+						[inputType]="'text'"
+						[value]="'from input value'"
+						[label]="'Your text with validate'"
+						[placeholder]="'Your placeholder'"
+						formControlName="text2">
+					</app-input>
+					<app-input [formControl]="inputTextDemoForm.controls['text3']" [disabled]="true" [inputType]="'text'" [label]="'Your text'" [placeholder]="'Your placeholder'" formControlName="text3"></app-input>
```

Comment:
don't need to keep `[formControl]` input since we already have `[formControlName]` which doing same thing